pkgs/events/src/emails/index.ts

```diff
@@ -0,0 +1,47 @@
+import { envs, isTest, l } from '@specfy/core';
+import { getResend, sendInvitation, sendWelcome } from '@specfy/emails';
+
+import { dispatcher } from '../client.js';
+
+export const resend = getResend(envs.RESEND_KEY || 'please_change_that_resend');
+
+export function consume() {
+  dispatcher.on('account.register', async ({ user }) => {
+    if (isTest) {
+      return;
+    }
+
+    l.info('Sending email', { to: user.email, type: 'welcome' });
+    await sendWelcome(
+      resend,
+      {
+        to: user.email,
+      },
+      { email: user.email, name: user.name }
+    );
+  });
+
+  dispatcher.on('invitation.created', async (obj) => {
+    if (isTest) {
```

Comment:
I don't find this elegant --putting `isTest()` everywhere. It should be handled via your local ENV VARs. Either I setup my services in local correctly or I don't set the VAR to disable it.
Here for example, I can't test the email service locally.
Also, there is a mix of `isTest()`, `isProd()`, `!isProd()`, `!isTest()`, one is probably enough.

Reply:
`isProd` and `isTest` are different, they are not opposite.
- isProd checks that we are indeed in production mode
- isTest checks that we are executing test 

That means isTest can still allow using the thing locally but not during test. Like emails in that case.
I'll clarify by moving the condition.